src/asprof.h

```diff
@@ -63,6 +63,30 @@ typedef struct {
 DLLEXPORT asprof_thread_local_data* asprof_get_thread_local_data(void);
 typedef asprof_thread_local_data* (*asprof_get_thread_local_data_t)(void);
 
+
+typedef int asprof_user_jfr_key;
```

Comment:
Maybe `unsigned int`?

Also, I'm not fully happy with "user" in this and other identifiers in this API (who is user?)
WDYT about `asprof_event_tag`? Alternative suggestions are welcome.

Reply:
It can be -1 in case of failure, like in "normal" C functions.

Reply:
user = the user of async-profiler (as opposed to the developer of async profiler = you).